neon_data_models/models/api/llm.py

```diff
@@ -165,8 +165,10 @@ def to_completion_kwargs(self, mq2role: dict = None) -> dict:
         history = self.messages[-2*self.max_history:]
         for msg in history:
             msg["role"] = mq2role.get(msg["role"]) or msg["role"]
-        history.insert(0, {"role": "system",
-                           "content": self.persona.system_prompt})
+        if self.persona.system_prompt:
```

Comment:
If system prompt is `None` it should be skipped
If system prompt is `""` it should add it as empty system prompt

Both cases are possible and different